app/stores/UsersStore.js

```diff
@@ -76,10 +87,45 @@ export default class UsersStore extends BaseStore<User> {
     invariant(res && res.data, "Data should be available");
     runInAction(`invite`, () => {
       res.data.users.forEach(this.add);
+      this.total += res.data.sent.length;
+      this.count.invited += res.data.sent.length;
+      this.count.all += res.data.sent.length;
```

Comment:
First time working with mobx – lmk if there's a better way of managing this observable